manifest.jps

```diff
@@ -79,6 +79,7 @@ onInstall:
       while [[ $(ps uax | grep "plesk" | grep -v grep | wc -l) = 0 ]] ; do sleep 2; done
       service mysql status || service mysql start
       plesk bin admin --set-admin-password -passwd "${globals.passwd}"
+      iptables -t nat -I PREROUTING -p tcp -m tcp ! -s 127.0.0.1/32 --dport 80 -j REDIRECT --to-ports 8443
```

Comment:
the redirect must be on Application Layer, I guess. You can't just direct unencrypted traffic via SSL port.